### PR TITLE
fix(e2e): quick fix to e2e test

### PR DESCRIPTION
**Pull request by iamhopaul123**
Fix broken e2e test after https://github.com/aws/amazon-ecs-cli-v2/pull/763.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: internal/pkg/cli/env_init.go [hunk 1]
```diff
@@ -152,7 +152,7 @@ func (o *initEnvOpts) Execute() error {
 	}
 
 	// 2. Get the environment
-	env, err := o.envDeployer.GetEnvironment(o.ProjectName(), o.EnvName)
+	env, err := o.envDeployer.GetEnvironment(o.ProjectName(), o.EnvName, o.IsProduction)
```
Comment (efekarakus) [inline internal/pkg/cli/env_init.go]: Why are we doing it this way? can we not use `ssm.GetEnvironment()`

Comment (iamhopaul123) [reply internal/pkg/cli/env_init.go]: at this stage it is not even written in SSM store...This is to write in SSM store.

Comment (efekarakus) [reply internal/pkg/cli/env_init.go]: oh i see 🤦‍♂ my bad!

How difficult would it be to modify the `templates/environment/cf.yml` template to accept a `IsProd` parameter that defaults to `"false"`? Modifying the `conf.ToEnv` seems to be a neater solution than passing around the bool.

If that's not possible, then I think I'd prefer:
```suggestion
	env, err := o.envDeployer.GetEnvironment(o.ProjectName(), o.EnvName)
	if err != nil {
		return fmt.Errorf("get environment struct for %s: %w", o.EnvName, err)
	}
	env.IsProduction = o.IsProduction
```

File: internal/pkg/cli/env_init.go [hunk 2]
```diff
@@ -156,6 +156,7 @@ func (o *initEnvOpts) Execute() error {
 	if err != nil {
 		return fmt.Errorf("get environment struct for %s: %w", o.EnvName, err)
 	}
+	env.Prod = o.IsProduction
```
Comment (efekarakus) [inline internal/pkg/cli/env_init.go]: can we add a unit test for this? where the GetEnvironment returns an env.Prod = false and we set it to true

**Pull request merged**